i18n/en.pot

```diff
@@ -527,6 +527,9 @@ msgstr ""
 msgid "Could not verify if this {{fieldDisplayName}} is unique"
 msgstr ""
 
+msgid "This fields is required"
```

Comment:
@HendrikThePendric should be "This field is required" or "These fields are required"